File: 프로그래머스/0/181879. 길이에 따른 연산/길이에 따른 연산.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) {
    int answer = 0;
    int now = 1;
    if (num_list.size() > 10) {
        for(int i: num_list) {
            answer += i;
        }
    } else {
        for(int i: num_list) {
            now *= i;
        }
        answer = now;
    }
    return answer;
}

File: 프로그래머스/0/181879. 길이에 따른 연산/길이에 따른 연산.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        int now=1;
        if(num_list.length >= 11) {
            for(int j: num_list) answer+=j;
        } else {
            for(int j: num_list) now*=j;
            answer = now;
        }
        return answer;
    }
}